diff --git a/lib/modules/EIP712/EIP712.utils.js b/lib/modules/EIP712/EIP712.utils.js
index 2aa2d39b63c4829623d32a0439c216a7214a9bee..aafde9bc8487c66fbe714ee5f86c24b001e5eb53 100644
--- a/lib/modules/EIP712/EIP712.utils.js
+++ b/lib/modules/EIP712/EIP712.utils.js
@@ -68,7 +68,7 @@ exports.getFiltersForMessage = exports.getSchemaHashForMessage = exports.sortObj
 var axios_1 = __importDefault(require("axios"));
 var sha224_1 = __importDefault(require("crypto-js/sha224"));
 var utils_1 = require("../../utils");
-var eip712_1 = __importDefault(require("@ledgerhq/cryptoassets/data/eip712"));
+var eip712_1 = { default: {} };
 var bignumber_js_1 = __importDefault(require("bignumber.js"));
 var NULL_ADDRESS = "0x0000000000000000000000000000000000000000";
 /**
diff --git a/lib/services/ledger/erc20.js b/lib/services/ledger/erc20.js
index 5d1643b9445dc5c9ac5eb98e9f6ea56da58e18d3..4c7328ff98f28538defb03a9e6fe3a45f7259981 100644
--- a/lib/services/ledger/erc20.js
+++ b/lib/services/ledger/erc20.js
@@ -42,7 +42,7 @@ exports.__esModule = true;
 exports.byContractAddressAndChainId = exports.findERC20SignaturesInfo = void 0;
 var axios_1 = __importDefault(require("axios"));
 var logs_1 = require("@ledgerhq/logs");
-var index_1 = require("@ledgerhq/cryptoassets/data/evm/index");
+var index_1 = { signatures: {} };
 var loadConfig_1 = require("./loadConfig");
 var asContractAddress = function (addr) {
     var a = addr.toLowerCase();
